PWGCF/FemtoDream/Core/femtoDreamUtils.h

```diff
@@ -121,6 +121,9 @@ inline float getMass(int pdgCode)
     case 3122: // Lambda
       Mass = o2::constants::physics::MassLambda;
       break;
+    case 4122: // Charm Lambda
+      Mass = o2::constants::physics::MassLambdaCPlus;
```

Comment:
```suggestion
      mass = o2::constants::physics::MassLambdaCPlus;
```
Names of variables start with lowercase letters.

Reply:
I am okay to make it in lowercase, but as you can see,  this mass variable in this task is defined with capital letters, is it advisable to modify it completely?

Reply:
Yes

Reply:
@vkucera I have implemented it, please have  a look if it looks okay

Reply:
Looks good, thanks.